docs/src/modules/components/MarkdownElement.js

```diff
@@ -479,16 +487,16 @@ const Root = styled('div')(
         flexShrink: 0,
         fontSize: '18px',
         margin: 'auto',
-        opacity: 0.6,
+        opacity: 0.4,
```

Comment:
The button feels easy to miss.

```suggestion
        opacity: 0.5,
```